CHANGELOG.md

```diff
@@ -233,6 +233,6 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 - Update caniuse to fix failed integration tests ([#2322](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2322))
 - Update caniuse to 1.0.30001460 to fix failed integration tests ([#3538](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3538))
-
+- Removed functions which are no longer in use([#3766](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3766))
```

Comment:
Our changelog process is confusing at the moment - but no need for a separate entry here.

```suggestion

```